### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by Amtresu**
I've gone ahead and added a fully implemented desktop version of the site. This is not only functional, but also stylized to match the template.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,4 +1,395 @@
+@media only screen and (min-width: 480px) {
```
Comment (cynthiainga) [inline style.css:1]: - [ ] Please kindly make sure you use the right `min-width`, as required in the project the `min-width` should be in the range of `768px` or wider :upside_down_face: :upside_down_face: 
- [ ] Your Desktop should look at the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) that you chose.
Your design

Screenshot from 2022-07-06 13-49-01

Figma design

Screenshot from 2022-07-06 13-49-33

File: index.html [hunk 2]
```diff
@@ -204,24 +219,144 @@ <h3 class="project1_title">Multi-Post Stories</h3>
...
+                <li class="language_descriptor">javaScript</li>
+            </ul>
         </div>
 
-        <div>
-            <p class="about_me">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button class="see_project" type="button">See Project</button>
+            </div>
+
+        </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] The same remarque about the desktop version, please kindly fix it for all the `project section` section :grin: :grin:. You need to center your card and arrange the button.
 
 Your design

Screenshot from 2022-07-06 13-51-23

Figma design

Screenshot from 2022-07-06 13-51-00

File: index.html [hunk 3]
```diff
@@ -204,24 +219,144 @@ <h3 class="project1_title">Multi-Post Stories</h3>
...
+
+    </section>
+
+
+    <section id="contact_info">
+        <div class="contact_info_content">
+            <h2 id="greeting_2">About<br>Myself</h2>
+        
+            <p class="about_me">I’m a software developer! I can help you build a<br> product , feature or website Look through some of my<br>work and experience! If you like what you see and have<br> a project you need coded, don’t hestiate to contact me.</p>
```
Comment (cynthiainga) [inline index.html:353]: - [ ] Please kindly fix also your `About Myself` section :upside_down_face: :upside_down_face: 

Your design

Screenshot from 2022-07-06 13-50-17
 
Figma design

Screenshot from 2022-07-06 13-50-39

File: index.html [hunk 4]
```diff
@@ -288,13 +421,44 @@ <h4>
...
+                <li><p class="slider_titles">Skills</p></li>
+                <li><img class="arrow2" src=img/right_arrow.png alt="right_arrow"></li>
+            </ul>
+
+            <img src="img/desktop_seperator.svg" class="last_seperator" alt="seperator">
+        </div>
+    </section>
+
         <section class="contact_form">
             <h3 class="contact_header">Contact Me</h3>
```
Comment (cynthiainga) [inline index.html:461]: - [ ] Please fix also you `contact-form` section for desktop :upside_down_face: :upside_down_face: 

Your design

Screenshot from 2022-07-06 13-51-52

Figma design

Screenshot from 2022-07-06 13-52-13

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [ ] No linters errors :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please kindly make your pull request more descriptive with more details
- [ ] Please kindly check the comments under the review for more details :point_down: :point_down: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Amtresu** [comment]
Hi @cynthiainga cynthiainga
solo


Sorry, just a couple of points of clarification.

I don't believe this is a pair-programming assignment, as for both days it's assigned its explicitly marked as solo. 
1440px

The figma design says its supposed to be designed for 1440 px width, so that is the width I designed it for.

**Comment by cynthiainga** [comment]
Ohh sorry, it's my error, I do a confusion sorry for that :pray: :pray:

**Comment by cynthiainga** [comment]
I already updated my required changes

**Comment by Amtresu** [comment]
@cynthiainga 

Thanks, 

I also see it looks like you are suggesting I design the website for 768px, which is a tablet size, I designed the website based upon the Figma templated, which is 1440px

**Comment by cynthiainga** [comment]
> @cynthiainga
> 
> Thanks,
> 
> I also see it looks like you are suggesting I design the website for 768px, which is a tablet size, I designed the website based upon the Figma templated, which is 1440px

@Amtresu 768px is the `min-width` and 1440px is the `max-width` for the desktop

**Comment by Amtresu** [comment]
@cynthiainga 
The assignment says here 
"Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider)."

It does not say it has to be designed for 768px size, just anything higher. So I went ahead and designed it for a normal screen size, like the template, and not a tablet.

thanks for bearing with me.

**Comment by Amtresu** [comment]
https://user-images.githubusercontent.com/101094809/177555034-076aed4a-570c-4e0e-a405-f3f81f30bdc9.mp4

**Comment by cynthiainga** [comment]
@Amtresu, I request you to use the `min-width 768px` for good practice and to help you for your project look good. There are some computers that have a small screen so it means that the person who uses the small screen or tablet can't check your portfolio :upside_down_face: cause the design is not adapted for their screen.
Or for example, if a recruiter opens your portfolio on their tablet what will happen if he/she sees that your portfolio is not adapted for all screen sizes!?
I'm just here to help you to improve your portfolio, to help you if I have some suggestions that can help to improve your project, I can give them to you, but if you do not agree with my changes required or my suggestions you can ask for the second opinion :+1: :+1:

**Review by Reem-lab — APPROVED** [review]
## Hi @Amtresu👋🏻👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights✨
- Good PR description ✅✅
- No linter errors ✅✅
- Good follow-up with the previous reviewer.✅✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**